base/compiler/ssair/inlining.jl

```diff
@@ -371,7 +371,6 @@ function ir_inline_item!(compact::IncrementalCompact, idx::Int, argexprs::Vector
             stmt′ = ssa_substitute!(idx′, stmt′, argexprs, sig, sparam_vals, linetable_offset, boundscheck, compact)
             if isa(stmt′, ReturnNode)
                 val = stmt′.val
-                isa(val, SSAValue) && (compact.used_ssas[val.id] += 1)
                 return_value = SSAValue(idx′)
                 inline_compact[idx′] = val
```

Comment:
`inline_compact[idx′] = val` calls `count_added_node!`, which increments `inline_compact.used_ssas`, which `=== compact.used_ssas`, so incrementing here is incorrect